tools/cli.py

```diff
@@ -309,6 +310,13 @@ def Main() -> None:
                 ('--attach_disks', 'Comma separated list of disk names '
                                    'to attach.', None)
             ])
+  AddParser('gcp', gcp_subparsers, 'deleteinstance', 'Delete a GCE instance.',
+            args=[
+                ('instance_name', 'Name of the GCE instance to delete.', ''),
+                ('--delete_disks',
+                 'Whethere to delete all disks attached to the instance.',
```

Comment:
typo Whethere -> Whether
and I think 'Whether to force delete all disks attached to the instance.' since it's only about deleting the RO and the 'don't delete when instance is deleted' disks. To no give the false impression that keeping this False will not delete attached disks

Reply:
agreed. Let me change this a bit to be more explicit